main.go

```diff
@@ -116,6 +116,10 @@ func main() {
 	if !*noUpdate {
 		updateCfg.Fetcher = updater.Fetcher(version.BuildVersion)
 	}
+	if version.BuildVersion == "dev" {
+		updateCfg.Fetcher = nil
+	}
```

Comment:
**Nit**: This could be more succintly represented with

```go
if version.BuildVersion != "dev" && !*noUpdate {
    updateCfg.Fetcher = updater.Fetcher(version.BuildVersion)
}
```

though maybe less readable that way.